scripts/update-roadmap-issues

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
+label="$1"
```

Comment:
Can we do a check for presence here and dump some usage info for anyone who runs it without args?